src/main/app/src/app/informatie-objecten/informatie-object-view/informatie-object-view.component.ts

```diff
@@ -58,6 +58,9 @@ export class InformatieObjectViewComponent extends ActionsViewComponent implemen
     historie: MatTableDataSource<HistorieRegel> = new MatTableDataSource<HistorieRegel>();
 
     historieColumns: string[] = ['datum', 'gebruiker', 'wijziging', 'oudeWaarde', 'nieuweWaarde', 'toelichting'];
+
+    zaakColumns: string[] = ['zaakID', 'zaakType', 'zaakStatus', 'zaakStartdatum', 'zaakStreefdatum', 'actions'];
```

Comment:
Ik zie niet zo snel waar dit voor is?

Reply:
klopt, zal ik weg halen. ik had de panels vervangen door een tabel. maar toch maar cards van gemaakt.